tests/dummy/app/pods/components/frost-create-form/component.js

```diff
@@ -0,0 +1,69 @@
+import Ember from 'ember'
+import layout from './template'
+
+export default Ember.Component.extend({
+  layout,
+  modalForms: Ember.inject.service('modal-forms'),
+  bunsenModel: {
+    'type': 'object',
+    'properties': {
+      'username': {
+        'type': 'string'
+      },
+      'description': {
+        'type': 'string'
+      },
+      'password': {
+        'type': 'string'
+      }
+    },
+    'required': [
+      'username', 'password'
+    ]
+  },
+
+  bunsenView: {},
+
+  willInsertElement () {
```

Comment:
Can you raise an issue to handle this more cleanly in the future?  I'd like the style of the overlay to not need this type of service to activate